app/filters/api/authentication_filter.rb

```diff
@@ -0,0 +1,16 @@
+module API
+  class AuthenticationFilter
+    MAS_CMS_API_TOKEN = ENV.fetch('MAS_CMS_API_TOKEN')
```

Comment:
What happens if MAS_CMS_API_TOKEN is `nil`. Should it raise an error or silently rejected request